src/utils/jsHelper/helper.tsx

```
@@ -380,18 +383,41 @@ export function getPreviewDetails(applicationData, documents) {
+		const decoded = decodeFromBase64(base64Str);
+		const parsed = JSON.parse(decoded); // assumes it's JSON
+		const [title] = parsed.credentialSchema.title.split(':');
+
+		return title.trim(); // handles both cases
+	} catch (error) {
+		console.error('Failed to extract title:', error);
+		return undefined;
+	}
+}
+
 export function getSubmmitedDoc(userData, document) {
```

ESLint

[error] 'document' is defined but never used.

(@typescript-eslint/no-unused-vars)